### ISSUE TITLE
Running linter on Windows does not respect import/order settings

### ISSUE THREAD

**Issue by thedavidprice** [benmosher/eslint-plugin-import#1725]
See https://github.com/redwoodjs/redwood/pull/395 

Example Windows lint errors here: 
https://github.com/redwoodjs/redwood/pull/395/checks?check_run_id=566887702

### Description
On Mac (local) and Ubuntu (GH Action runner), the linter correctly applies import/order settings. The following import lines  _pass_:
```javascript
// web/src/RedwoodProvider.js

import PropTypes from 'prop-types'

import { GraphQLProvider as RealGraphQLProvider } from 'src/graphql'
```
On Windows, both locally and via GH Actions runner, the code above results in a lint error:
```shell
D:\a\redwood\redwood\packages\web\src\RedwoodProvider.js
##[error]  1:1  error  There should be no empty line within import group  import/order
```

### Config
Import/order settings are here:
https://github.com/redwoodjs/redwood/blob/05bfc65a775190f9ba9238a06441741cd44574fb/packages/eslint-config/index.js#L96-L109

```javascript
'import/order': [
      'error',
      {
        groups: [
          'builtin',
          'external',
          'internal',
          'parent',
          'sibling',
          'index',
        ],
        'newlines-between': 'always',
      },
    ],
```

**Comment by darekkay** [comment]
Which version are you using?
Seems to be related: https://github.com/benmosher/eslint-plugin-import/issues/1692

**Comment by thedavidprice** [comment]
@darekkay We were at v2.20.0 Just changed to v2.20.2 and 🎉 

Huge help. Thanks for checking this and asking.
